src/test/java/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEvalIntegrationTest.java

```diff
@@ -106,6 +112,55 @@ public void testFundamentalsCountVariantsSNPsAndIndelsWithNovelty() throws IOExc
         spec.executeTest(name, this);
     }
 
+    @DataProvider(name = "testContigStratWithUserSuppliedIntervalsData")
+    public Object[][] testContigStratWithUserSuppliedIntervalsData() {
+        final SAMSequenceDictionary expectedDictionary = ReferenceUtils.loadFastaDictionary(new File(b37Reference.replace("fasta.gz", "dict")));
+        Set<String> allContigs = new TreeSet<>(expectedDictionary.getSequences().stream().map(SAMSequenceRecord::getSequenceName).collect(Collectors.toSet()));
+
+        List<Object[]> tests = new ArrayList<>();
+        tests.add(new Object[]{"1:1-1480226", Collections.singletonList("1")});
+        tests.add(new Object[]{"1", Arrays.asList("1")});
+        tests.add(new Object[]{null, allContigs});
+        tests.add(new Object[]{"2", Collections.singletonList("2")});
+        return tests.toArray(new Object[][]{});
+    }
```

Comment:
The original version of these tests was much easier to understand - it just needed a data provider, which for the old version of the tests would just be:

```
    @DataProvider(name = "testContigStratWithIntervals")
    public Object[][] testContigStratWithIntervals() {
        return new Object[][] {
                { "testContigStratWithUserSuppliedIntervals", "-L 1:1-1480226" },
                { "testContigStratWithUserSuppliedIntervals2", "-L 1" },
                { "testContigStratWithUserSuppliedIntervals3", null },
        };
    }

```

Reply:
discussed below - made the change